UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -303,18 +308,66 @@ public void DestroyItem(ObjectBehaviour item, HashSet<GameObject> alreadySold)
 			export.Count += count;
 			export.TotalValue += credits;
 
+			var itemAttributes = item.GetComponent<ItemAttributesV2>();
+			if (itemAttributes)
+			{
+				foreach (var itemTrait in itemAttributes.GetTraits())
```

Comment:
can you reformulate this logic to reduce nesting? Maybe use ``Validations.HasItemTrait`` instead of iterating all traits or even extract some portions into private methods.